### PR TITLE
Added new feature to draw routes with multiple points

### PR DESCRIPTION
**Pull request by ArystanK**

### CODE REVIEW

File: maps-kt-compose/src/commonMain/kotlin/centre/sciprog/maps/compose/MapFeature.kt [hunk 1]
```diff
@@ -40,6 +40,18 @@ abstract class MapCustomFeature(
     abstract fun drawFeature(drawScope: DrawScope, offset: Offset)
 }
 
+class MapRouteFeature(
```
Comment (altavir) [inline maps-kt-compose/src/commonMain/kotlin/centre/sciprog/maps/compose/MapFeature.kt:43]: Couldn't it be replaced by combination of lines? Or it has some unique capabilities?

**Comment by altavir** [comment]
I've made some breaking changes to the core features API, could you please resolve conflicts?

**Comment by ArystanK** [comment]
Unnecessary functionality

**Comment by altavir** [comment]
@ArystanK I think that we actually want this feature, especially if you add additional parameters like PointMode and caps to the feature definition. We can call it `route` instead of `points` though. Could you resolve conflicts with the current branch?

**Pull request closed without merging**